### ISSUE TITLE
Could you please upload the CT-RATE and NLST datasets used for pretraining?

### ISSUE THREAD

**Issue by jsadu826** [Luffy03/Large-Scale-Medical#13]

**Comment by Luffy03** [comment]
hi nlst is already uploaded. Ct-rate requires license and approval thus you need to refer to the original link

**Comment by jsadu826** [comment]
Sorry, I could not find nlst either in prect-160k or voco_downstream on huggingface.

**Comment by Luffy03** [comment]
Ohhhhh I forgot it. Thanks for your reminder. NLST is too big and it takes time to upload it. Since my server has been suspended recently, I have to upload it a few weeks later. Or you can download it from the original source.

**Comment by jsadu826** [comment]
Please upload it if possible later. Or upload the preprocessing script, since NLST is originally in dicom format, not nii. Great thanks!

**Comment by Luffy03** [comment]
Yes, we curate and preprocess the NLST dataset. I will check this part as soon as possible but uploading this dataset would be a bit longer. Or I can check and upload the preprocess scripts. Thank you very much for your understanding!

**Comment by Luffy03** [comment]
Dear jsadu, I start uploading NLST to https://huggingface.co/datasets/Luffy503/PreCT-160K/tree/main/NLST. I am sorry for my late reply, since it requires 4.6 TB of storage and it takes a really long time to zip these files. And I am not sure how long it would take to upload these files to huggingface.

**Comment by jsadu826** [comment]
Great thanks! I notice there's already a 32G NLST_convert_v100 file on huggingface, but this is only part of the dataset, right?

**Comment by Luffy03** [comment]
Yes, there would be 156 files, each with 32G ...... still a long way to go

**Comment by jsadu826** [comment]
Looking forward to it!

**Comment by Luffy03** [comment]
Dear jsadu, all NLST files are uploaded now. Please check. Thanks for your attention again.

**Comment by jsadu826** [comment]
Dear author, there seems only 134 files on huggingface, the NLST_convert_v19026 ~ NLST_convert_v19035 and NLST_convert_v19047 ~ NLST_convert_v19056 are missing.

**Comment by Luffy03** [comment]
So sorry that I am too busy to deal with it recently. I will solve it as soon as possible.

**Comment by Luffy03** [comment]
> Dear author, there seems only 134 files on huggingface, the NLST_convert_v19026 ~ NLST_convert_v19035 and NLST_convert_v19047 ~ NLST_convert_v19056 are missing.

Dear jsadu, these files are uploaded, you could take a look if you are interested.

**Comment by jsadu826** [comment]
Great thanks!

**Comment by junwenliu-imit** [comment]
Dear author, I tried to first concatenate all files using commands like "cat NLST_convert_v1* >> NLST_all", then decompress it using "tar -xzvf NLST_all", but errors occurred. How should I correctly decompress the whole dataset?

![Image](https://github.com/user-attachments/assets/782d0ddc-8ca8-4191-aaeb-5d6a913ad643)

**Comment by jsadu826** [comment]
Dear author, I tried to first concatenate all files using commands like "cat NLST_convert_v1* >> NLST_all", then decompress it using "tar -xzvf NLST_all", but errors occurred. How should I correctly decompress the whole dataset?

![Image](https://github.com/user-attachments/assets/a86c1e4e-44e0-43d8-911c-159d6a50da66)

**Comment by Luffy03** [comment]
It may be caused by unsuccessful downloads or the version of decompression toolkit.
